package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "build": "rollup -c --configPlugin @rollup/plugin-typescript",
     "dev": "pnpm run build -w",
-    "deploy": "adb shell am start -S -n com.discord/.MainActivity",
+    "deploy": "pnpm run build && adb push dist/Aliucord.js.bundle /sdcard/AliucordRN/Aliucord.js.bundle && adb shell am start -S -n com.discord/.MainActivity",
```

Comment:
&& does not work on windows

Reply:
wdym? it does

Reply:
~~yarn makes it work on windows~~ how else are you supposed to do that then